src/index/script/HeadsUpDisplay/_HeadsUpDisplay.coffee

```diff
@@ -10,5 +10,6 @@ export default class HeadsUpDisplay extends React.Component
 	render: =>
 		<div id="headsupdisplay">
```

Comment:
Nit - I would camelCase the 'headsUpDisplay' for improved readability.

Reply:
Good idea -- done!